apm-sniffer/apm-sdk-plugin/clickhouse-0.3.2.x-plugin/src/main/resources/skywalking-plugin.def

```diff
@@ -0,0 +1,17 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+clickhouse-0.3.2.x=org.apache.skywalking.apm.plugin.jdbc.clickhouse.v32.define.ConnectionInstrumentation
```

Comment:
The plugin of 0.3.x uses `clickhouse-0.3.x`. As 0.3.2 is an incompatible change, that keys should be updated to `clickhouse-0.3.1`.

Reply:
clickhouse-0.3.x-plugin to clickhouse-0.3.1-plugin.